lib/ruby_lsp/requests/support/syntax_tree_formatting_runner.rb

```diff
@@ -25,8 +25,11 @@ def initialize
             )
         end
 
-        sig { params(_uri: String, document: Document).returns(T.nilable(String)) }
-        def run(_uri, document)
+        sig { params(uri: String, document: Document).returns(T.nilable(String)) }
+        def run(uri, document)
+          relative_path = Pathname.new(URI(uri).path).relative_path_from(Pathname.new(URI(RubyLsp::WORKSPACE_URI).path))
```

Comment:
Instead of parsing the `WORKSPACE_URI` on every request, can we update the constant to store the `URI` object and then update the usage on diagnostics?

Reply:
fixed https://github.com/Shopify/ruby-lsp/pull/670/commits/4ab54f9dab60c7539e567d21952920f4e582e837